### ISSUE TITLE
Import multiple queries on client

### ISSUE THREAD

**Issue by micchickenburger** [Swydo/meteor-graphql#20]
It doesn't seem that I can rename `query` on import.  For instance, if I try the following:

    import testQuery from './test.graphql';

or

    import { query as testQuery } from './test.graphql';

I receive the following error:

```
Exception in defer callback: TypeError: Cannot read property 'kind' of undefined
    at checkDocument (http://localhost:3000/packages/modules.js?hash=401da39174ff7894f4c0f56bdb7e1fd90ca15cb0:1213:13)
    at getQueryDefinition (http://localhost:3000/packages/modules.js?hash=401da39174ff7894f4c0f56bdb7e1fd90ca15cb0:1253:5)
    at QueryManager.watchQuery (http://localhost:3000/packages/modules.js?hash=401da39174ff7894f4c0f56bdb7e1fd90ca15cb0:4215:31)
    at ApolloClient.watchQuery (http://localhost:3000/packages/modules.js?hash=401da39174ff7894f4c0f56bdb7e1fd90ca15cb0:4798:34)
    at Blaze.TemplateInstance.gqlQuery (http://localhost:3000/packages/swydo_blaze-apollo.js?hash=7915f676b5c961fc380b475b07e249564591b4b2:250:28)
    at Blaze.View.<anonymous> (http://localhost:3000/app/app.js?hash=25bc50a15f463632fc27b8e1e3858cb6fb3a4cc0:2277:26)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1934:20
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3744:12)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1932:29
    at Object.Blaze._withCurrentView (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:2271:12)
```

The consequence of this is that I can only import one query per JavaScript document.  I know that I can do comment includes of other graphql files; however, this doesn't help me on the client where each query accepts different variables.

**Comment by jamiter** [comment]
Hi! This package leverages the `graphql-tag` package where they are working on implementing this feature. See https://github.com/apollographql/graphql-tag/pull/122 for more info.

As soon as they added it in a new release, I'll make sure this package will be updated to!

**Comment by jamiter** [comment]
@micchickenburger, this is now available in 0.4.0!

You can now import multiple queries like this:

```graphql
query foo { foo }

query bar { bar }
```

```js
import { foo, bar } from './your-graphql-file.graphql';
```

See https://github.com/apollographql/graphql-tag/pull/122

Let me know if you run into any issues!
